### ISSUE TITLE
new version of Text::Wrap on CPAN

### ISSUE THREAD

**Issue by p5pRT** [Perl/perl5#9673]
Migrated from [rt.perl.org#63672](https://rt-archive.perl.org/perl5/Ticket/Display.html?id=63672) (status was 'resolved')

Searchable as RT63672$

**Comment by p5pRT** [comment]
### From @muir

#### Created by muir@idiom.org



I've released a new version of Text&#8203;::Wrap to CPAN\.  Please update
the version that ships with perl\.

Changes&#8203;:

Test improvements from Dave Mitchel sent back in 2005\.\.\.

Added code to increase $columns if it's not big enough to accommodate
the subsequent tab\.

Minor documentation fixes from David Landgren \<david at landgren\.net>\.

Use warnings&#8203;::warnif instead of just warn for columns \< 2\.  Appled per
request of Rafael Garcia\-Suarez \<rgarciasuarez at gmail\.com>\.

<details><summary>Perl Info</summary>

```

Flags:
    category=library
    severity=low

Site configuration information for perl 5.10.0:

Configured by Debian Project at Thu May  8 13:32:42 UTC 2008.

Summary of my perl5 (revision 5 version 10 subversion 0) configuration:
  Platform:
    osname=linux, osvers=2.6.24.4, archname=i486-linux-gnu-thread-multi
    uname='linux ninsei 2.6.24.4 #1 smp preempt fri apr 18 15:36:09 pdt 2008 i686 gnulinux '
    config_args='-Dusethreads -Duselargefiles -Dccflags=-DDEBIAN -Dcccdlflags=-fPIC -Darchname=i486-linux-gnu -Dprefix=/usr -Dprivlib=/usr/share/perl/5.10 -Darchlib=/usr/lib/perl/5.10 -Dvendorprefix=/usr -Dvendorlib=/usr/share/perl5 -Dvendorarch=/usr/lib/perl5 -Dsiteprefix=/usr/local -Dsitelib=/usr/local/share/perl/5.10.0 -Dsitearch=/usr/local/lib/perl/5.10.0 -Dman1dir=/usr/share/man/man1 -Dman3dir=/usr/share/man/man3 -Dsiteman1dir=/usr/local/man/man1 -Dsiteman3dir=/usr/local/man/man3 -Dman1ext=1 -Dman3ext=3perl -Dpager=/usr/bin/sensible-pager -Uafs -Ud_csh -Ud_ualarm -Uusesfio -Uusenm -DDEBUGGING=-g -Doptimize=-O2 -Duseshrplib -Dlibperl=libperl.so.5.10.0 -Dd_dosuid -des'
    hint=recommended, useposix=true, d_sigaction=define
    useithreads=define, usemultiplicity=define
    useperlio=define, d_sfio=undef, uselargefiles=define, usesocks=undef
    use64bitint=undef, use64bitall=undef, uselongdouble=undef
    usemymalloc=n, bincompat5005=undef
  Compiler:
    cc='cc', ccflags ='-D_REENTRANT -D_GNU_SOURCE -DDEBIAN -fno-strict-aliasing -pipe -I/usr/local/include -D_LARGEFILE_SOURCE -D_FILE_OFFSET_BITS=64',
    optimize='-O2 -g',
    cppflags='-D_REENTRANT -D_GNU_SOURCE -DDEBIAN -fno-strict-aliasing -pipe -I/usr/local/include'
    ccversion='', gccversion='4.2.3 (Debian 4.2.3-5)', gccosandvers=''
    intsize=4, longsize=4, ptrsize=4, doublesize=8, byteorder=1234
    d_longlong=define, longlongsize=8, d_longdbl=define, longdblsize=12
    ivtype='long', ivsize=4, nvtype='double', nvsize=8, Off_t='off_t', lseeksize=8
    alignbytes=4, prototype=define
  Linker and Libraries:
    ld='cc', ldflags =' -L/usr/local/lib'
    libpth=/usr/local/lib /lib /usr/lib /usr/lib64
    libs=-lgdbm -lgdbm_compat -ldb -ldl -lm -lpthread -lc -lcrypt
    perllibs=-ldl -lm -lpthread -lc -lcrypt
    libc=/lib/libc-2.7.so, so=so, useshrplib=true, libperl=libperl.so.5.10.0
    gnulibc_version='2.7'
  Dynamic Linking:
    dlsrc=dl_dlopen.xs, dlext=so, d_dlsymun=undef, ccdlflags='-Wl,-E'
    cccdlflags='-fPIC', lddlflags='-shared -O2 -g -L/usr/local/lib'

Locally applied patches:
    


@INC for perl 5.10.0:
    /etc/perl
    /usr/local/lib/perl/5.10.0
    /usr/local/share/perl/5.10.0
    /usr/lib/perl5
    /usr/share/perl5
    /usr/lib/perl/5.10
    /usr/share/perl/5.10
    /usr/local/lib/site_perl
    .


Environment for perl 5.10.0:
    HOME=/home/muir
    LANG=en_US
    LANGUAGE (unset)
    LD_LIBRARY_PATH (unset)
    LOGDIR (unset)
    PATH=.:/home/muir/bin/sh:/home/muir/bin:/home/muir/bin/share:/bin:/usr/bin:/sbin:/usr/sbin:/usr/local/shbin:/usr/local/sbin:/usr/local/bin:/usr/local/ptybin:/usr/X11R6/bin:/usr/bin/X11:/usr/local/tex/bin:/usr/ucb:/usr/bin:/bin:/etc:/usr/etc:/usr/games:/lib:/usr/lib:/usr/local/java/bin:/usr/lib/uucp:/usr/openwin/bin:/usr/openwin/bin/xview:/usr/openwin/demo:/usr/adm:/home/muir/tmp
    PERL_BADLANG (unset)
    SHELL=/bin/tcsh

```
</details>

**Comment by p5pRT** [comment]
### From @tsee

Thanks\, the module was updated in core with commit
69e34dac306d4c474199dd63fa07c93e2e08570a

**Comment by p5pRT** [comment]
The RT System itself - Status changed from 'new' to 'open'

**Comment by p5pRT** [comment]
### From nj88udd02@sneakemail.com

\(David Muir Sharnoff\) \(via RT\) wrote&#8203;:
> \# New Ticket Created by  \(David Muir Sharnoff\) 
> \# Please include the string&#8203;:  \[perl \#63672\]
> \# in the subject line of all future correspondence about this issue\. 
> \# \<URL&#8203;: http&#8203;://rt\.perl\.org/rt3/Ticket/Display\.html?id=63672 >

> I've released a new version of Text&#8203;::Wrap to CPAN\.  Please update
> the version that ships with perl\.

Thanks\, the module has been updated in core with commit 
69e34dac306d4c474199dd63fa07c93e2e08570a\.

To whom it may concern&#8203;: This ticket can be closed\.

Cheers\,
Steffen

**Comment by p5pRT** [comment]
### From tchrist@perl.com

>\(David Muir Sharnoff\) \(via RT\) wrote&#8203;:
>> \# New Ticket Created by  \(David Muir Sharnoff\) 
>> \# Please include the string&#8203;:  \[perl \#63672\]
>> \# in the subject line of all future correspondence about this issue\. 
>> \# \<URL&#8203;: http&#8203;://rt\.perl\.org/rt3/Ticket/Display\.html?id=63672 >

>> I've released a new version of Text&#8203;::Wrap to CPAN\.  Please update
>> the version that ships with perl\.

>Thanks\, the module has been updated in core with commit 
>69e34dac306d4c474199dd63fa07c93e2e08570a\.

>To whom it may concern&#8203;: This ticket can be closed\.

I'm pretty disappointed on its UTF\-8 ignorance\.

Since it refused to try to get it right\, it 
\*should\* die if passed utf8 data\.

The manpage note about the other module is little help\.

\-\-tom

**Comment by p5pRT** [comment]
### From p5p@spam.wizbit.be

On Thu Mar 05 23&#8203;:20&#8203;:53 2009\, muir@&#8203;idiom\.org wrote&#8203;:
>
> I've released a new version of Text&#8203;::Wrap to CPAN\.  Please update
> the version that ships with perl\.

In the future it might be easier to just send a mail to perl5\-
porters@&#8203;perl\.org instead of submitting a bug report via perlbug\.


Kind regards\,

Bram

**Comment by p5pRT** [comment]
p5p@spam.wizbit.be - Status changed from 'open' to 'resolved'
